process/block/shardblock_test.go

```diff
@@ -2563,6 +2594,14 @@ func TestShardProcessor_ReceivedMetaBlockShouldRequestMissingMiniBlocks(t *testi
 		requestHandler,
 		&mock.PreProcessorContainerMock{},
 		&mock.InterimProcessorContainerMock{},
+		&mock.RequestedItemsHandlerMock{
+			HasCalled: func(key string) bool {
+				return false
```

Comment:
This is not really tested

Reply:
It is tested, because Has method returns false and the request is executed